src/views/v2/events-bar/form.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * View: Events Bar Form
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/events-bar/form.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+
+$current_url = tribe_events_get_current_filter_url();
+?>
+<div class="tribe-events-calendar-events-bar--form">
+
+	<form action="<?php echo esc_attr( $current_url ); ?>" method="post" class="tribe-common-c-search">
```

Comment:
The form will move out. The whole View will be the form.
Check here: https://github.com/moderntribe/the-events-calendar/pull/2496/files#diff-a94dec2f05fa33e5969767bcecff4943R34

Reply:
If I'm understanding you correctly, Gustavo, you're saying that this should be all one view and not multiple pieces (using `$this->template()` below)? This is my feel, that this search (form) should be one view, as we even have a search component defined in `tribe-common`. That should have all the styles for the form.